com.polydez.KeyVault.yml

```diff
@@ -0,0 +1,36 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/master/data/flatpak-manifest.schema.json
+app-id: com.polydez.RivalTune
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: rivaltune
+separate-locales: false
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --share=ipc
+  - --device=dri
+
+
+modules:
+  - name: rivaltune
+    buildsystem: simple
+    build-commands:
+      - unzip rivaltune-linux.zip
+      - install -d /app/rivaltune/lib
+      - install -d /app/rivaltune/data
+      - install -d /app/bin
+      - install -Dm755 rivaltune-linux/rivaltune -t /app/rivaltune/
+      - cp -r rivaltune-linux/lib/* /app/rivaltune/lib/
+      - cp -r rivaltune-linux/data/* /app/rivaltune/data/
+      - ln -s /app/rivaltune/rivaltune /app/bin/rivaltune
+      - install -Dm644 rivaltune-linux/rivaltune.png /app/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png
+      - install -Dm644 rivaltune-linux/com.polydez.RivalTune.desktop /app/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm644 rivaltune-linux/com.polydez.RivalTune.metainfo.xml /app/share/metainfo/${FLATPAK_ID}.metainfo.xml
+      
+    sources:
+      - type: file
+        dest-filename: rivaltune-linux.zip
+        url: https://github.com/polydezcom/RivalTune/releases/download/v1.2.0/rivaltune-v1.2.0-release.zip
+        sha256: 3adfe55f7574e14938548c19e606c6a96bb96ed321216a7d3f078cde51cf2dd0
```

Comment:
this should be built from source https://github.com/flatpak/flatpak-builder-tools?tab=readme-ov-file#flutter